scripts/generate-tokens-wrapper.js

```diff
@@ -123,55 +124,66 @@ const removeImporantComments = (css) => {
     return css.replaceAll(importantCommentRegExp, '');
 };
 
-const processTokens = (srcPath) => {
+const processTokens = (srcPath, tokensDir) => {
     let css = fs.readFileSync(srcPath, 'utf8');
     const fileName = srcPath.split(path.sep + 'css' + path.sep).at(-1);
     css = removeImporantComments(targetHost(css));
 
+    // s2 doesn't need express tokens
+    if (tokensDir === 'tokens-v2' && fileName.startsWith('express')) {
+        return;
+    }
+
     fs.writeFileSync(
-        path.join(__dirname, '..', 'tools', 'styles', 'tokens', fileName),
+        path.join(__dirname, '..', 'tools', 'styles', tokensDir, fileName),
         css
     );
 };
 
-const processPackages = async (srcPath, index) => {
+const processPackages = async (srcPath, tokensDir, index) => {
     const packageName = tokenPackages[index];
-    const expressPath = path.join(srcPath, 'express.css');
     const spectrumPath = path.join(srcPath, 'spectrum.css');
 
-    // check if expressPath exists
-    if (fs.existsSync(expressPath)) {
-        let express = fs.readFileSync(expressPath, 'utf8');
-        express = removeImporantComments(targetHost(express));
+    // check if spectrumPath exists
+    if (fs.existsSync(spectrumPath)) {
+        let spectrum = fs.readFileSync(spectrumPath, 'utf8');
+        spectrum = removeImporantComments(targetHost(spectrum));
         fs.appendFileSync(
             path.join(
                 __dirname,
                 '..',
                 'tools',
                 'styles',
-                'tokens',
-                'express',
+                tokensDir,
+                'spectrum',
                 'global-vars.css'
             ),
-            express
+            spectrum
         );
     }
 
-    // check if spectrumPath exists
-    if (fs.existsSync(spectrumPath)) {
-        let spectrum = fs.readFileSync(spectrumPath, 'utf8');
-        spectrum = removeImporantComments(targetHost(spectrum));
+    // spectrum-2 doesn't need express package tokens
+    if (tokensDir === 'tokens-v2') {
+        return;
+    }
+
+    const expressPath = path.join(srcPath, 'express.css');
+
+    // check if expressPath exists
+    if (fs.existsSync(expressPath)) {
```

Comment:
This is repetition. Instead abstract this logic to a method called `processPackage` which takes in the packageType as `spectrum | express`
```js
const processPackage = async (srcPath, tokensDir, index, packageType) => {
    const packageFile = packageType === 'spectrum' ? 'spectrum.css' : 'express.css';
    const packagePath = path.join(srcPath, packageFile);

    if (fs.existsSync(packagePath)) {
        let content = fs.readFileSync(packagePath, 'utf8');
        content = removeImporantComments(targetHost(content));
        const outputPath = path.join(
            __dirname,
            '..',
            'tools',
            'styles',
            tokensDir,
            packageType,
            'global-vars.css'
        );
        fs.appendFileSync(outputPath, content);
    }
};
```

And call the above like this
```js
 await processPackage(srcPath, tokensDir, index, 'spectrum');
 ```